internal/providers/docker/compose/project_test.go

```diff
@@ -173,10 +598,24 @@ func TestParseService(t *testing.T) {
 			}
 
 			comp, err := compose.Parse(&content)
-			assert.NoError(t, err)
+			if !assert.NoError(t, err) {
+				return
+			}
 
 			svc := comp.Services["test-svc"]
 			assert.Equal(t, expected, svc)
 		})
 	}
 }
+
+func boolRef(b bool) *bool {
```

Comment:
I've seen functions like these in so many projects and it's such a bummer that they are necessary :-P

Reply:
Yep! It would be really nice if there was a generic `HeapAlloc(T) -> *T` function :D